### ISSUE TITLE
Return to the previous web page using the B and Y buttons on the controller.

### ISSUE THREAD

**Issue by MikhailGribov** [Igalia/wolvic#533]
In the standard browser on pico 4 you can go back using the B and Y buttons. Can you bind these buttons also in the wolvic browser?

**Comment by svillar** [comment]
OK, if it's a platform wide feature I think it does make sense

**Comment by svillar** [comment]
Apparently in Meta they're used also to perform the go back action in the system (although interestingly not in the browser :man_shrugging: )

Anyway, @javifernandez I think we'll play safe if we implement that for all of our supported architectures (with controllers) and not only for Pico

**Comment by javifernandez** [comment]
> Apparently in Meta they're used also to perform the go back action in the system (although interestingly not in the browser man_shrugging )
> 

It seems that Meta Browser performs a "word selection + contextual menu" when pressing the Y and B button.

**Comment by svillar** [comment]
> > Apparently in Meta they're used also to perform the go back action in the system (although interestingly not in the browser man_shrugging )
> 
> It seems that Meta Browser performs a "word selection + contextual menu" when pressing the Y or B buttons.

Meta Browser does that indeed, but in the rest of the applications I tried they all do back. Perhaps you could research a bit more about that, but it looks to me that the browser is following its own path, and is not being consistent with the rest of the platform.

**Comment by javifernandez** [comment]
> Meta Browser does that indeed, but in the rest of the applications I tried they all do back. 

Yes, Desktop apps, especially the ones pre-installed and that use the system's UI, are consistent on using the buttons for controlling the navigation (eg Files). 

However, there is the angle of considering these buttons are contextual actions, which behavior is tied to the nature and purpose of the application. In this sense, having a consistent behavior between apps of the same category (browsers in this case) seems important as well. 

Perhaps it'd be interesting to investigate why Meta Browser implemented that behavior.
